### ISSUE TITLE
Account for different screen sizes.

### ISSUE THREAD

**Issue by pwightman** [ZuseApp/zuse-ios#82]
We talked through a lot of options here, but never settled on anything. The basic idea is that we want projects to be created natively on any screen size, and played on any screen size. It involves some amount of encoding the screen size it was created on into the zuse file format, so that when it's read in by another device, it can scale it properly, perhaps adding black box on the sides/top if the device it was created on has a different aspect ratio than the one it's running on (e.g. Created on iPhone 4S, attempting to play on iPhone 5/6/6+)

**Comment by pwightman** [comment]
Actually, you know what... if we remove ZuseHub, there would be no way to share projects. So maybe we don't have to completely solve this before 1.0. We should at the very least embed the screen size it was created on into the file format (since that's hard to correct after release) but we don't have to worry about actually scaling it (which was the hard part). Thoughts @mhogenso?

**Comment by ghost** [comment]
Yes.  I agree.  If I remember correctly the screen size is already getting stored in the file.  I'll have to double check though.
